shanoir-ng-studies/src/main/java/org/shanoir/ng/subject/dto/mapper/SubjectDecorator.java

```diff
@@ -18,12 +18,18 @@ public class SubjectDecorator implements SubjectMapper {
 	public PageImpl<SubjectDTO> subjectsToSubjectDTOs(Page<Subject> page) {
 		Page<SubjectDTO> mappedPage = page.map(new Function<Subject, SubjectDTO>() {
 			public SubjectDTO apply(Subject entity) {
-				return subjectToSubjectDTO(entity);
+				return subjectToSubjectDTONoStudies(entity);
 			}
 		});
 		return new PageImpl<>(mappedPage);
 	}
 
+	@Override
+	public SubjectDTO subjectToSubjectDTONoStudies(Subject subject) {
+		final SubjectDTO subjectDTO = delegate.subjectToSubjectDTONoStudies(subject);
+		return subjectDTO;
```

Comment:
I think you can directly return `delegate.subjectToSubjectDTONoStudies(subject);` here

Reply:
done